_posts/2020-03-18-cockpit-215.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cockpit 215
+author: kkoukiou
+date: 2020-03-18
+tags: cockpit linux
+slug: cockpit-215
+category: release
+summary: Cockpit with Network improvements and Docker partial deprecation
```

Comment:
Let's drop the "partial" here. Deprecation already means that we don't immediately drop it everywhere.